lib/shopify_cli.rb

```diff
@@ -117,6 +117,7 @@ module ShopifyCli
   autoload :ProjectType, 'shopify-cli/project_type'
   autoload :Resources, 'shopify-cli/resources'
   autoload :SubCommand, 'shopify-cli/sub_command'
+  autoload :Shopifolk, 'shopify-cli/shopifolk'
```

Comment:
Let's move this up to keep the alphabetical ordering.

Reply:
you can see that i was 1 line off from a genuine attempt